content/rc/api/examples/manage-subscriptions.md

```diff
@@ -44,26 +44,56 @@ POST "https://[host]/v1/subscriptions"
 }
 ```
 
-To use the sample JSON document in your own account, you must modify these parameters:
+To use the sample JSON document in your own account, modify these parameters:
 
-- **`paymentMethodId`** - Specify a payment method that is defined for your account.
+- **`paymentMethodId`** - Specify a payment method that is connected to your account.
```

Comment:
I would say either: 
...connected to your account. 
or
...for your account.
"that is connected to" is too many words